piksi_tools/console/settings_view.py

```diff
@@ -212,6 +227,40 @@ def _factory_default_button_fired(self):
                         + "Are you sure you want to reset to factory defaults?"
     confirm_prompt.run(block=False)
 
+  def _settings_save_to_file_fired(self):
```

Comment:
Is it going to be obvious that you're savings the current Piksi settings to a file, not the current set of settings in the console (that may or may not have been saved to Piksi's flash)?